### PR TITLE
Send mail notify exam created more 8h

### PR DESCRIPTION
**Pull request by hoangmirs**

### CODE REVIEW

**Comment by phandinhman** [comment]
Ok!

File: config/deploy.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Location: config/deploy.rb
+namespace :delayed_job do
+  desc "Start delayed_job process"
+  task :start, :roles => :app do
```
Comment (dieunb) [inline config/deploy.rb]: `roles: :app`

File: config/deploy.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    run "cd #{current_path}; RAILS_ENV=#{rails_env} script/delayed_job start"
+  end
+
+  desc "Stop delayed_job process"
+  task :stop, :roles => :app do
```
Comment (dieunb) [inline config/deploy.rb]: `roles: :app`

File: config/deploy.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    run "cd #{current_path}; RAILS_ENV=#{rails_env} script/delayed_job stop"
+  end
+
+  desc "Restart delayed_job process"
+  task :restart, :roles => :app do
```
Comment (dieunb) [inline config/deploy.rb]: ` roles: :app`

**Pull request merged**